### DISCUSSION TITLE
"undefined" in html- button- response (not going away on pavlovia)

### DISCUSSION THREAD

**Discussion by ncuneo0711** [jspsych/jsPsych#2038] [Q&A] [answered]
Hi! I am having an issue when I run my study via pavlovia. Locally- the code is working perfectly but when I run in through pavlovia for some reason there's an unidentified "undefined" that appears on the screen. 

I tried it with the example code to make sure it wasn't my code that was the issue. 

I used this example:
var mturk_code = {
    type: 'html-button-response',
    stimulus: '<p>Running</p>',
    choices: ['Healthy', 'Unhealthy'],
    prompt: "<p>Is this activity healthy or unhealthy?</p>"
};

And this is what it looks like (see screenshot below) when I run it through pavlovia (not locally. Locally it is working fine!). The same thing is happening on my personal code.

Any thoughts? 

<img width="523" alt="Screen Shot 2021-08-03 at 2 04 21 PM" src="https://user-images.githubusercontent.com/65300058/128064444-e86d7145-f245-4b8e-8124-00f88bbb8880.png">

**Comment by ncuneo0711** [comment, marked as answer]
I can't figure out how to delete this but found the answer!!

**Reply by becky-gilbert** [reply]
Thanks for the update! Feel free to post the answer here, in case it's helpful for others 😃
